IdentityCore/src/cache/mac/MSIDMacTokenCache.h

```diff
@@ -22,8 +22,29 @@
 // THE SOFTWARE.
 
 #import <Foundation/Foundation.h>
-#import <MSIDTokenCacheDataSource.h>
+#import "MSIDTokenCacheDataSource.h"
+
+@class MSIDMacTokenCache;
+
+@protocol MSIDMacTokenCacheDelegate <NSObject>
+
+- (void)willAccessCache:(nonnull MSIDMacTokenCache *)cache;
```

Comment:
I'd love to see some tests checking that delegate calls are invoked?

Reply:
added